### ISSUE TITLE
:FzfLua files opens file in insert mode

### ISSUE THREAD

**Issue by 9seconds** [ibhagwan/fzf-lua#46]
Thanks for the plugin!

Each time I open a file via `:FzfLua files` or its Lua counterpart (with split or with Enter), it opens in insert mode. Can I disable it somehow? I would like to open a file in a normal mode instead.

**Comment by ibhagwan** [comment]
Definitely irritating and not by design, can you share a bit more details about your setup so I can try to reproduce this (OS, neovim version, etc)?

**Comment by 9seconds** [comment]
I've managed to find out an offensive setting in my own config. Here it is:

```vim
augroup CorrectCTRLDForTerminal
  autocmd!
  autocmd TermClose * call feedkeys("i")
augroup END
```

I've made this one to avoid irritating press-any-key-to-quit in terminal if I finish it with Ctrl+D. Apparently, this sent `i` after fzf window has been closed. I haven't expected that autocmd will work in that case.

So, I've switched feedkeys to send Esc and the problem is gone.
